src/_includes/banner.html

```diff
@@ -1,32 +1,38 @@
-<section class="usa-banner" aria-label="Official government website">
+<section
+  class="usa-banner"
+  aria-label="Official website of the United States government"
+>
   <div class="usa-accordion">
     <header class="usa-banner__header">
       <div class="usa-banner__inner">
         <div class="grid-col-auto">
           <img
+            aria-hidden="true"
             class="usa-banner__header-flag"
             src="{{ '/assets/img/us_flag_small.png' | url }}"
-            alt="U.S. flag"
+            alt=""
           />
         </div>
-        <div class="grid-col-fill tablet:grid-col-auto">
+        <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
           <p class="usa-banner__header-text">
             An official website of the United States government
           </p>
-          <p class="usa-banner__header-action" aria-hidden="true">
-            Here’s how you know
-          </p>
+          <p class="usa-banner__header-action">Here’s how you know</p>
         </div>
         <button
+          type="button"
           class="usa-accordion__button usa-banner__button"
           aria-expanded="false"
-          aria-controls="gov-banner-default"
+          aria-controls="gov-banner-default-default"
         >
           <span class="usa-banner__button-text">Here’s how you know</span>
         </button>
       </div>
     </header>
-    <div class="usa-banner__content usa-accordion__content" id="gov-banner-default">
+    <div
+      class="usa-banner__content usa-accordion__content"
+      id="gov-banner-default-default"
```

Comment:
@mahoneycm curious as to the extra `default` here. Were there conflicts with other banners?

Reply:
Oh yikes this is actually in the [component code example](https://designsystem.digital.gov/components/banner/) on our site, I just copied and pasted and potentially missed this? I'll try to dive deeper on when and why this change happened

Reply:
Thanks for looking into that. In the mean time can we just remove the extra one here?